src/mm-rating/mm-rating.html

```diff
@@ -0,0 +1,29 @@
+<link rel="import" href="../../bower_components/polymer/polymer.html"/>
+<link rel="import" href="../mm-icon/mm-icon.html"/>
+
+<dom-module id="mm-rating">
+    <link rel="import" type="css" href="mm-rating.css"/>
+    <template>
+        <div class="rating-wrapper">
+            <template is="dom-if" if="{{isDisplayTotals(displayTotals)}}">
```

Comment:
displayTotals is already a boolean so there is no need for the method passthrough here